BUILD.bazel

```diff
@@ -229,12 +229,22 @@ alias(
     visibility = ["//visibility:public"],
 )
 
-cc_binary(
+alias(
     name = "protoc",
```

Comment:
Does this make it impossible to opt-out?  If that's the case, we should probably hold off on this until 30.x at least

Reply:
Can we enable this with some sort of flag, and flip the default in a subsequent breaking change if desired instead?

Reply:
Yes, it's possible to add a completely Starlark flag, to control if prebuilt binaries are used or not.